mobile/src/common/types/navigation/navigation-item.type.ts

```diff
@@ -7,6 +8,7 @@ type NavigationItem = {
   permissions: PermissionKey[];
   component: React.FC;
   isAuthRequired: boolean;
+  screenOptions?: DrawerNavigationOptions;
```

Comment:
This is drawer navigation related only options, so they shouldn't be in NavigationItem, move it to DrawerNavigationItem